www/include/eventLogs/xml/data.php

```diff
@@ -616,20 +640,33 @@
     }
 
     $limitReq = "";
-    $limitReq2 = "";
     if ($export !== "1") {
-        $limitReq = " LIMIT " . $num * $limit . ", " . $limit;
+        $offset = $num * $limit;
+        $queryValues['offset'] = [\PDO::PARAM_INT => $offset];
+        $limitReq = " LIMIT :offset, :limit";
     }
-    $dbResult = $pearDBO->query($req . $limitReq);
-    $rows = $pearDBO->query("SELECT FOUND_ROWS()")->fetchColumn();
+    $stmt = $pearDBO->prepare($req . $limitReq);
+    foreach ($queryValues as $bindId => $bindData) {
+        foreach ($bindData as $bindType => $bindValue) {
+                $stmt->bindValue($bindId, $bindValue, $bindType);
+        }
+    }
+
+    $stmt->execute();
 
-    if (!($dbResult->rowCount()) && ($num != 0)) {
+    if (!($stmt->rowCount()) && ($num != 0)) {
         if ($export !== "1") {
-            $limitReq2 = " LIMIT " . (floor($rows / $limit) * $limit) . ", " . $limit;
+            $offset = floor($rows / $limit) * $limit;
         }
-        $dbResult = $pearDBO->query($req . $limitReq2);
+        $stmt->bindValue('offset', $offset, PDO::PARAM_INT);
```

Comment:
Why two different calculation methods for the same condition? `if ($export !== "1")`
line 644: `$offset = $num * $limit;`
here: `$offset = floor($rows / $limit) * $limit;`

Reply:
As discussed together @callapa this will not be changed for now as the calculation methodology for the pagination is quite different for each file. Larger subject that will be adressed in an other ticket